src/controls/Jalousie.ts

```diff
@@ -27,9 +46,13 @@ export class Jalousie extends ControlBase {
         await this.createBooleanControlStateObjectAsync(control.name, uuid, control.states, 'up', 'indicator', {
             write: true,
         });
+        this.addStateEventHandler(control.states.up, this.upDownChangeHandler.bind(this), 'up');
```

Comment:
I know, `bind()` is used a lot in JavaScript, but it is ugly IMHO.

Please simply use an arrow function like so:
```js
this.addStateEventHandler(control.states.up, () => this.upDownChangeHandler(), 'up');
````

Same below.

Reply:
Well, neither notation is particularly elegant and I actually prefer the `bind`, but as you wish.